### DISCUSSION TITLE
Errors while attempting render of mesh data

### DISCUSSION THREAD

**Discussion by krak3n1013** [jneilliii/OctoPrint-BedLevelVisualizer#574] [Q&A]
TypeError: Cannot set properties of null (setting '0')

Why am I getting this error and how do I fix it?

**Comment by jneilliii** [comment]
where and when are you getting that error? might need to enable debug logging on the support tab, restart octoprint and try again and that can help diagnose better with the plugin_bedlevelvisualiver_debug.log file.

**Reply by krak3n1013** [reply]
I think I might have figured out the problem. I think I have the wrong gcode on the config page. I'm going to change the gcode and try another mesh.
